docs/user/guides/best-practice/index.rst

```diff
@@ -0,0 +1,36 @@
+How-to guides: best practice
+============================
+
+Over the years,
+we have become familiar with a number of methods that work well and which we consider **best practice**.
+
+⏩️ :doc:`Best practices for linking to your documentation </automatic-redirects>`
+    Documentation changes over time,
+    and links and cross-references can become challenging manage for various reasons.
+    Here is a set of best practices explaining and addressing these challenges.
+
+⏩️ :doc:`Deprecating content </guides/deprecating-content>`
+    Best practice for removing or deprecating documentation content.
+
+⏩️ :doc:`Create reproducible builds </guides/reproducible-builds>`
```

Comment:
```suggestion
⏩️ :doc:`Creating reproducible builds </guides/reproducible-builds>`
```

We should choose a tense and stick to it.

Reply:
Agreed, the form of writing "Create" was from when we removed "[How to] Create ...", so it came from a navigation style that's no longer relevant.